OneCommunityGlobal/HighestGoodNetworkApp
# Jay - PERMISSION: Edit Time Entry Description Permission

# Description
This PR enables the user with the Edit Time Entry Description permission to be able to access any user's time entry description and edit it and update successfully.

## Related PRS (if any):
This frontend PR is related to the [#712](https://github.com/OneCommunityGlobal/HGNRest/pull/712) backend PR.

…

## Main changes explained:
- Added a new permission label for Edit Time Entry Description permission.
- Added conditional logic so that the user with the permission can view the edit button and edit the time log description.

## How to test (please watch the video):
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log in with an owner account
5. go to Other Links -> Permissions Management -> Manage User Permissions -> Choose a user without Edit Time Entry Description permission under Timelog Management, Edit Time Entries, Edit Timelog Description -> click on Add  -> Scroll down then submit
6. verify that the user who was given the permission is now able to see the the edit button on another user's dashboard for time log entries.

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118868270/3dc44aab-1f81-4f26-8c97-09f82afe301c


## Note:
**Unmute the video**
